13/umbraco-deploy/deploy-settings.md

```diff
@@ -37,7 +37,6 @@ For illustration purposes, the following structure represents the full set of op
             "ApiKey": "<your API key here>",
             "ApiSecret": "<your API secret here>",
             "Edition": "Default",
-            "DefaultTimeoutSeconds": 60,
```

Comment:
This isn't a configurable setting, but a read-only constant field with the default timeout used by other settings...

Reply:
ah, makes sense! should I remove this setting in all other deploy versions as well?

Reply:
and would it perhaps make better sense to remove this setting and add a warning or info note at the end of the "timeout settings" https://docs.umbraco.com/umbraco-deploy/v/12.deploy.latest/deploy-settings#timeout-settings that "DefaultTimeoutSeconds is set to default of 60 seconds. This isn't a configurable setting, but a read-only constant field with the default timeout used by other settings"?

Reply:
`DefaultTimeoutSeconds` should be removed from the documentation for all versions, because it's not a setting at all and because it's an internal constant, also isn't available externally (besides actually being set to `20 * 60`, so 20 minutes, which is already documented in the timeout settings section).